x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/dot_expander.test.tsx

```diff
@@ -77,9 +77,28 @@ describe('Processor: Dot Expander', () => {
 
     // Expect form error as "field" does not contain '.'
     expect(form.getErrorsMessages()).toEqual([
-      'A field value requires at least one dot character.',
+      'The field value should contain at least one dot character or equal to an asterisk.',
     ]);
   });
+
+  test('allow form submission if field for the dot notation equal to *', async () => {
```

Comment:
Thanks for adding a test!

Reply:
Show appropriate default description of dot expander when wildcard is used : [3a25cd5](https://github.com/elastic/kibana/pull/123522/commits/3a25cd5fd046de3bc47a9834c2f934acb8bb4e3b)

Change related to default description is made. :slightly_smiling_face: